src/components/List/AddList/AddList.js

```diff
@@ -39,55 +39,58 @@ const AddList = () => {
   };
 
   return (
-    <>
-      <Collapse isOpen={open}>
-        <div className="m-1 w-64">
-          <input
-            type="text"
-            ref={inputRef}
-            value={title}
-            placeholder="Type something..."
-            autoFocus
-            className="w-full text-left box-border outline-none border-2 border-trello-blue-100 shadow bg-white hover:bg-trello-gray-400 rounded-md p-2 my-1.5"
-            // onBlur={() => setOpen(false)}
-            onChange={handleOnChange}
-            onKeyPress={handleKeyPress}
-          />
-          <div className="flex items-center">
-            <button
-              type="button"
-              className="py-2 px-3 bg-trello-green-100 hover:bg-trello-green-200 text-white transition ease-in duration-200 text-center text-base shadow-md rounded-md"
-              onClick={handleAddList}
-            >
-              Add List
-            </button>
-            <FiX
-              onClick={handleClose}
-              size={36}
-              className="text-trello-gray-200 hover:bg-trello-gray-500 cursor-pointer rounded-full ml-1 p-1"
+    <div className= { open && "shadow bg-trello-gray-100 rounded-md w-80 m-1 px-2 py-2 flex flex-col list-wrapper" } >
+      {open ? 
```

Comment:
This will stop the other div from being rendered which affected the height of the container.